### PR TITLE
network and volume managment with libvirt

### PR DESCRIPTION
**Pull request by goneri**

### CODE REVIEW

File: virt_lightning/configuration.py [hunk 1]
```diff
@@ -9,11 +8,12 @@
 
 DEFAULT_CONFIGURATION = {
     "main": {
-        "libvirt_uri": "qemu:///system" if os.geteuid() == 0 else "qemu:///session",
```
Comment (zloidemon) [inline virt_lightning/configuration.py]: Is script require to run by root now? Why did you remove session?

File: virt_lightning/virt_lightning.py [hunk 2]
```diff
@@ -214,53 +235,73 @@ def kvm_binary(self):
...
-        else:
-            raise Exception("Unsupported libvirt URI")
+            self.storage_pool_obj = self.create_storage_pool(
+                name=storage_pool, directory=storage_dir
+            )
 
-        full_dir = storage_dir / "upstream"
         try:
-            full_dir.mkdir(parents=True, exist_ok=True)
+            full_dir = self.get_storage_dir() / "upstream"
```
Comment (zloidemon) [inline virt_lightning/virt_lightning.py]: Looks like bug. `str` can not divide another `str`.

Comment (goneri) [reply virt_lightning/virt_lightning.py]: / is a specialized operator of PosixPath. It is used to build paths.

File: README.md [hunk 3]
```diff
@@ -103,9 +120,15 @@ sudo usermod -a -G kvm,libvirt $USER
 ## Installation
 
 ```shell
-python3 -m venv venv
-source venv/bin/activate
-pip install git+https://github.com/virt-lightning/virt-lightning
+pip3 install --prefix ~/.local git+https://github.com/virt-lightning/virt-lightning
```
[the remark this answers is not in the document]
Comment (goneri) [reply README.md]: Pip already uses this path by default on Ubuntu and Debian when you do a pip install as a user. I believe npm does too.

File: README.md [hunk 4]
```diff
@@ -103,9 +120,15 @@ sudo usermod -a -G kvm,libvirt $USER
 ## Installation
 
 ```shell
-python3 -m venv venv
```
[the remark this answers is not in the document]
Comment (goneri) [reply README.md]: All the deps are installed from the RPM.

File: README.md [hunk 5]
```diff
@@ -103,9 +120,15 @@ sudo usermod -a -G kvm,libvirt $USER
...
-source venv/bin/activate
-pip install git+https://github.com/virt-lightning/virt-lightning
+pip3 install --prefix ~/.local git+https://github.com/virt-lightning/virt-lightning
+```
+
+`virt-lightning` will be installed in ~/.local/bin/. Add it in your `$PATH` if
+it's not already the case. For instance if you use:
+
+```shell
+echo "export PATH=$PATH:~/.local/bin/" >> ~/.bashrc
```
[the remark this answers is not in the document]
Comment (goneri) [reply README.md]: Well, they can still do ~/.local/bin/vl?

File: virt_lightning/configuration.py (hunk 1 above)
Comment (zloidemon) [reply virt_lightning/configuration.py]: Talked at chat about that.

File: README.md (hunk 4 above)
[the remark this answers is not in the document]
Comment (goneri) [reply README.md]: This being said, we can still explain how to install it in a venv like before. I just try to find the easiest to deploy the app.
